enqueue/fs/0.4/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "ENQUEUE_DSN": "file:/%VAR_DIR%/enqueue"
```

Comment:
I don't know about Linux, but on macOS `file:///var/www/dev/enqueue` not only works, but it's the only way this can work: 3 `/` after `file:` is normal.